iocAdmin/Db/genSiteEnvVars.pl

```diff
@@ -0,0 +1,9 @@
+print("file iocEnvVar.template\n");
+print("{\n");
+print("pattern\n");
+print("{ ENVNAME, ENVVAR, ENVTYPE }\n");
```

Comment:
Using a here-doc could reduce the number of quotes and escaped newlines, and I recommend always quoting the argument to `file` (again, not in the original). My personal preference for formatting substitution files would have fewer newlines:
```perl
print << __END__;
file "iocEnvVar.template" {
  pattern { ENVNAME, ENVVAR, ENVTYPE }
__END__
```